### PR TITLE
Aws guard duty

### PR DESCRIPTION
**Pull request by LenaigKaliou**
Still need to add some particular cases (s.a. anomalous behaviour or evidences) and tests
Still need to add smart descriptions
Some fields doesn't always work (geo fields)

### CODE REVIEW

**Comment by LenaigKaliou** [comment]
Suppression of useless fields
Correction of typos
Some tests were added

**Comment by LenaigKaliou** [comment]
Added smart descriptions
Added a new test
Added a delete field to avoid two same data in different fields

File: AWS/aws-guardduty/tests/event_ekscluster.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                },
+                {
+                  "key": "GeneratedFindingEKSClusterTag2",
+                  "value": "GeneratedFindingEKSClusterTagValue2"
+                },
+                {
+                  "key": "GeneratedFindingEKSClusterTag3",
+                  "value": "GeneratedFindingEKSClusterTagValue3"
+                }
+              ]
```
Comment (squioc) [inline AWS/aws-guardduty/tests/event_ekscluster.json:88]: I think we can remove this field

**Review by squioc — CHANGES_REQUESTED** [review]

**Comment by LenaigKaliou** [comment]
Fields removed
Threat fields enriched
1 new test for ecs cluster

**Review by squioc — CHANGES_REQUESTED** [review]
Nice work :+1: 
Here are some comments.

**Comment by LenaigKaliou** [comment]
Changes made
No user_agent.full field, I used user_agent.original but it might be user_agent.os.full (it's not very clear with my logs)
I am not sure that threat.enrichments.indicator is necessary, I put those fields directly in threat.indicator
Change of syntax for files, paths and hashes, put in flattened list for more clarity
I only let the anomalies in additional infos for unusual behavior

File: AWS/aws-guardduty/tests/event_container.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        "name": "SuspiciousFile"
+      },
+      "enrichments": [
+        {
+          "indicator": {
+            "first_seen": "2023-03-20T15:33:12.000Z",
+            "last_seen": "2023-03-20T15:33:12.000Z",
+            "sightings": "1",
+            "type": "file",
+            "file_1": {
```
Comment (squioc) [inline AWS/aws-guardduty/tests/event_container.json:43]:
```suggestion
            "file": {
```

File: AWS/aws-guardduty/tests/event_container.json [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+              "name": "eicar.com",
+              "path": "tmp/eicar.com",
+              "hash": "275a021bbfb6489e54d471899f7db9d1663fc345ec2fe2a2c4538aabf651fd0f"
+            },
+            "file_2": {
```
Comment (squioc) [inline AWS/aws-guardduty/tests/event_container.json:48]: Should be in another indicator

File: AWS/aws-guardduty/tests/event_container.json [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        {
+          "indicator": {
+            "first_seen": "2023-03-20T15:33:12.000Z",
+            "last_seen": "2023-03-20T15:33:12.000Z",
+            "sightings": "1",
+            "type": "file",
+            "file_1": {
+              "name": "eicar.com",
+              "path": "tmp/eicar.com",
+              "hash": "275a021bbfb6489e54d471899f7db9d1663fc345ec2fe2a2c4538aabf651fd0f"
```
Comment (squioc) [inline AWS/aws-guardduty/tests/event_container.json:46]:
```suggestion
              "hash": { "sha256": "275a021bbfb6489e54d471899f7db9d1663fc345ec2fe2a2c4538aabf651fd0f"}
```

File: AWS/aws-guardduty/tests/event_container.json [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+            },
+            "file_2": {
+              "name": "eicar-2.txt",
+              "path": "tmp/eicar-2.txt",
+              "hash": "275a021bbfb6489e54d471899f7db9d2363fc345ec2fe2a2c4538aabf651ad0x"
```
Comment (squioc) [inline AWS/aws-guardduty/tests/event_container.json:51]:
```suggestion
              "hash":  { "sha256": "275a021bbfb6489e54d471899f7db9d2363fc345ec2fe2a2c4538aabf651ad0x"}
```

**Review by squioc — CHANGES_REQUESTED** [review]

**Comment by LenaigKaliou** [comment]
Enrichments fixed

**Pull request merged**